test/e2e_node/garbage_collector_test.go

```diff
@@ -175,7 +201,7 @@ func containerGCTest(f *framework.Framework, test testRun) {
 					for i := 0; i < pod.numContainers; i++ {
 						containerCount := 0
 						for _, containerName := range containerNames {
-							if strings.Contains(containerName, pod.getContainerName(i)) {
+							if containerName == pod.getContainerName(i) {
```

Comment:
Have you tested this?  I dont remember why I used `Contains` initially, but if it works the new way, I guess that is better.

Reply:
Yeah, because container name in `docker ps` is pretty long, it's composed of a lot of things.

Reply:
Oh, so because this is using `Labels[types.KubernetesContainerNameLabel]`, it should be exactly the name we gave it, rather than an element of the name.  I like it

Reply:
Yeah